### DISCUSSION TITLE
你知道神經網路和深度學習是怎麼運作的嗎？

### DISCUSSION THREAD

**Discussion by dabbie109303501** [1101-GS4719-C/Python-Community-GS4719-C#711] [科技新知]
一樣是之前推薦過的頻道3Blue1Brown
覺得第一部影片把深度學習講解的很清楚明瞭
其他部影片雖然難了一些 但也是用簡單的字句在描述 不會很艱澀難懂
推薦給大家
https://youtube.com/playlist?list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi

**Comment by ruthchen1020** [comment]
感謝分享，真的講解的很清楚👍

**Comment by danwan1027** [comment]
感謝分享，學習到了很多

**Comment by williamyen0623** [comment]
萬事起頭難，能有淺顯易懂的大綱解說真的對學習很有幫助，謝謝分享

**Comment by manipulateed** [comment]
哇 這個分享對於要踏入深度學習的小白 太友善了 謝謝🙏。

**Comment by Yunhans** [comment]
雖然看了但還是一知半解，好難

**Comment by AnAnistired** [comment]
這頻道好讚喔，感謝分享，科學英文的影片報告有著落了😍

**Reply by dabbie109303501** [reply]
我也超喜歡這個頻道ㄉ

**Comment by Pietra1226** [comment]
謝謝分析，原來是這樣，看來我還有很多要學的⋯

**Comment by a45320614** [comment]
3Blue1Brown是我很喜歡的頻道，透過圖形化及化繁為簡的描述，完美地傳達了科普的精神

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
呃 不知道是不是我的問題 看完還是不怎麼了解

**Comment by CPCpyt** [comment]
感謝分享，算是對這個部分有更深的認識。
很好奇這樣的技術如何擴展至其他未知的領域。
